TrackingProduction/Fun4All_TrackSeeding.C

```diff
@@ -0,0 +1,432 @@
+            << std::endl;
+
+  string outDir = "myKShortReco/";
+  string outputFileName = "outputKFParticle_" + to_string(runnumber) + "_" + to_string(segment) + ".root";
+  string outputRecoDir = outDir + "inReconstruction/";
+  string outputRecoFile = outputRecoDir + outputFileName;
+
+  if(doKFParticle){
+    string makeMainDirectory = "mkdir -p " + outDir;
+    system(makeMainDirectory.c_str());
+    string makeDirectory = "mkdir -p " + outputRecoDir;
+    system(makeDirectory.c_str());
+  }
+
+  // distortion calibration mode
+  /*
+   * set to true to enable residuals in the TPC with
+   * TPC clusters not participating to the ACTS track fit
+   */
+  G4TRACKING::SC_CALIBMODE = false;
+  TRACKING::pp_mode = true;
+
+  ACTSGEOM::mvtxMisalignment = 100;
+  ACTSGEOM::inttMisalignment = 100.;
+  ACTSGEOM::tpotMisalignment = 100.;
+  TString outfile = outfilename + "_" + runnumber + "-" + segment + ".root";
+  std::string theOutfile = outfile.Data();
+  auto se = Fun4AllServer::instance();
+  se->Verbosity(1);
+
+
+  Fun4AllRunNodeInputManager *ingeo = new Fun4AllRunNodeInputManager("GeoIn");
+  ingeo->AddFile(geofile);
+  se->registerInputManager(ingeo);
+
+  CDBInterface *cdb = CDBInterface::instance();
+  std::string tpc_dv_calib_dir = cdb->getUrl("TPC_DRIFT_VELOCITY");
+  if (tpc_dv_calib_dir.empty())
+  {
+    std::cout << "No calibrated TPC drift velocity for Run " << runnumber << ". Use default value " << G4TPC::tpc_drift_velocity_reco << " cm/ns" << std::endl;
+  }
+  else
+  {
+    CDBTTree *cdbttree = new CDBTTree(tpc_dv_calib_dir);
+    cdbttree->LoadCalibrations();
+    G4TPC::tpc_drift_velocity_reco = cdbttree->GetSingleFloatValue("tpc_drift_velocity");
+    std::cout << "Use calibrated TPC drift velocity for Run " << runnumber << ": " << G4TPC::tpc_drift_velocity_reco << " cm/ns" << std::endl;
+  }
+
+  G4TPC::ENABLE_MODULE_EDGE_CORRECTIONS = true;
+  // to turn on the default static corrections, enable the two lines below
+  //G4TPC::ENABLE_STATIC_CORRECTIONS = true;
+  // G4TPC::DISTORTIONS_USE_PHI_AS_RADIANS = false;
+
+  G4MAGNET::magfield_rescale = 1;
+  TrackingInit();
+
+  auto hitsinclus = new Fun4AllDstInputManager("ClusterInputManager");
+  hitsinclus->fileopen(inputclusterRawHitFile);
+  se->registerInputManager(hitsinclus);
+
+  // reject laser events if G4TPC::REJECT_LASER_EVENTS is true
+  Reject_Laser_Events();
+
+  /*
+   * Begin Track Seeding
+   */
+
+  /*
+   * Silicon Seeding
+   */
+
+  /*
+  auto silicon_Seeding = new PHActsSiliconSeeding;
+  silicon_Seeding->Verbosity(0);
+  silicon_Seeding->searchInIntt();
+  silicon_Seeding->setinttRPhiSearchWindow(0.4);
+  silicon_Seeding->setinttZSearchWindow(1.6);
+  silicon_Seeding->seedAnalysis(false);
+  se->registerSubsystem(silicon_Seeding);
+  */
+
+  auto silicon_Seeding = new PHActsSiliconSeeding;
+  silicon_Seeding->Verbosity(0);
+  // these get us to about 83% INTT > 1
+  silicon_Seeding->setinttRPhiSearchWindow(1.0);
```

Comment:
the rphi search window should be 0.4 and the z search window should be 2. We probably need to readjust these given the new alignment, but that is a later task